.github/workflows/release.yml

```diff
@@ -20,14 +20,14 @@ jobs:
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         
       - name: Build Zip
         id: build-zip
         run: |
           TAG=$(echo $GITHUB_REF | awk -F '/' '{ print $3}' | sed 's/v//')
           echo $TAG
-          echo "::set-output name=tag::$TAG"
+          echo "{tag}={$TAG}" >> $GITHUB_OUTPUT
```

Comment:
Since we are not using the TAG anymore, it would be better to just remove lines 28-30

Reply:
Good point. Removed in [08b8c20](https://github.com/flowforge/installer/pull/74/commits/08b8c20f131f2ecb0eddd43482f62046474066aa)